my-vars/config-sample-do-not-edit.yml

```diff
@@ -8,6 +9,32 @@
 # suitable.
 ###############################################################################
 
+###############################################################################
+# Server Setup
+#
+# Ansible script to setup and later to update portal server
```

Comment:
This comment doesn't make sense. I'm not sure what we are trying to convey to the portal operator here.

Reply:
Replaced with comments above.